index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> C3 Python BootCamp | Ghana 2022</title>
+  <link rel="stylesheet" href="./static/css/style.css">
+  <script src="https://kit.fontawesome.com/b40153832f.js" crossorigin="anonymous"></script>
+  <script src="./static/js/main.js" type="module"></script>
+</head>
+<body>
+  <header>
+    <nav id="top-nav">
+      <ul>
+        <li class="social-media">
+          <i class="fab fa-facebook"></i>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>Logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
+    </nav>
+    <nav id="menu-nav">
+      <div class="mobile-navbar">
+        <div id="hamburger-menu">
+          <button type="button" id="open-hamburger" class="button-none"><i class="fa fa-bars"></i></button>
+          <button type="button" id="close-hamburger" class="button-none"><i class="fas fa-times"></i></button>
+        </div>
+        <ul class="nav-links">
+          <li>
+            <a href="./about.html">About</a>
+          </li>
+          <li>
+            <a href="#">Program</a>
+          </li>
+          <li>
+            <a href="#">Join</a>
+          </li>
+          <li>
+            <a href="#">Sponsor</a>
+          </li>
+          <li>
+            <a href="#">News</a>
+          </li>
+        </ul>
+      </div>
+      <div class="large-navbar">
+        <a href="./index.html" class="logo">
+          <img  src="./static/images/logo/logo.png" alt="C3 logo">
+        </a>
+        <ul class="nav-links">
+          <li>
+            <a href="./about.html">About</a>
+          </li>
+          <li>
+            <a href="#">Program</a>
+          </li>
+          <li>
+            <a href="#">Join</a>
+          </li>
+          <li>
+            <a href="#">Sponsor</a>
+          </li>
+          <li>
+            <a href="#">News</a>
+          </li>
+        </ul>
+        <div class="campaign">
+          <div class="campaign-button">
+            C3 Campaign
+          </div>
```

Comment:
Why don't we make this `nav` a fixed position, in order to make your page look nice and not cover up important details on your website? 🤔 

![Screenshot (310)](https://user-images.githubusercontent.com/66112760/142416752-2b194803-dac1-45f3-a183-39572b70e6c9.png)